src/validator.ts

```diff
@@ -602,25 +627,31 @@ export class RegExpValidator {
     private onFlags(
```

Comment:
Should we rename this function to `onRegExpFlags` as well?

```suggestion
    private onRegExpFlags(
```

Reply:
Thanks for checking this out. Sounds good. I think the change would be more consistent 👍